drivers/pwm/pwm_mspm0.c

```diff
@@ -137,6 +155,197 @@ static int mspm0_pwm_get_cycles_per_sec(const struct device *dev,
 	return 0;
 }
 
+#ifdef CONFIG_PWM_CAPTURE
+#define MSPM0_CTRCTL_CAC_CCCTL_ACOND(x) (x << 10)
+
+static void mspm0_set_combined_mode(const struct pwm_mspm0_config *config,
+				    struct pwm_mspm0_data *data)
+{
+	DL_Timer_setLoadValue(config->base, data->period);
+	DL_Timer_setCaptureCompareInput(config->base, 0,
+			((config->cc_idx[0] & 0x1) ?
+			  DL_TIMER_CC_IN_SEL_CCPX : DL_TIMER_CC_IN_SEL_CCP0),
+			config->cc_idx[0]);
+
+	DL_Timer_setCaptureCompareInput(config->base, 0,
+				GPTIMER_IFCTL_01_ISEL_CCPX_INPUT_PAIR,
```

Comment:
indent so that it goes under last '(' + 1 space. If your editor is configured to follow linux code style, it should be automatic.

Apply that everywhere where relevant